R/slide.R

```diff
@@ -88,62 +88,85 @@
 #'     before = 1, as_list_col = TRUE
 #'   ) %>%
 #'   ungroup()
-epi_slide <- function(x, f, ..., before = NULL, after = NULL, ref_time_values = NULL,
-                      new_col_name = "slide_value", as_list_col = FALSE,
-                      names_sep = "_", all_rows = FALSE) {
+epi_slide <- function(
+    x, f, ...,
+    .window_size = 0, .align = c("right", "center", "left"), .ref_time_values = NULL,
+    .new_col_name = "slide_value", .as_list_col = FALSE, .names_sep = "_", .all_rows = FALSE,
+    before = NULL, after = NULL, ref_time_values = NULL, new_col_name = NULL,
+    as_list_col = NULL, names_sep = NULL, all_rows = NULL) {
+  if (any(map(c(n, before, after, ref_time_values, new_col_name, as_list_col, names_sep, all_rows), Negate(is.null)))) {
```

Comment:
suggest: use something like `%in% rlang::call_args_names(rlang::call_match())` so if they try to use it as tidyeval they get a better error message.

issue: if not using above suggestion, `map` should be `map_lgl`